x-pack/plugin/sql/jdbc/src/test/java/org/elasticsearch/xpack/sql/jdbc/VersionParityTests.java

```diff
@@ -38,14 +38,19 @@ public void testExceptionThrownOnIncompatibleVersions() throws IOException, SQLE
     }
     
     public void testNoExceptionThrownForCompatibleVersions() throws IOException {
-        prepareResponse(null);
-        
-        String url = JdbcConfiguration.URL_PREFIX + webServerAddress();
-        try {
-            new JdbcHttpClient(JdbcConfiguration.create(url, null, 0));
-        } catch (SQLException sqle) {
-            fail("JDBC driver version and Elasticsearch server version should be compatible. Error: " + sqle);
-        }
+        Version version = Version.CURRENT;
+        do {
+            prepareResponse(version);
+
+            String url = JdbcConfiguration.URL_PREFIX + webServerAddress();
```

Comment:
Those can be outside of the loop no?
and probably the whole try - catch can "wrap" the loop?

Reply:
Yes, you're right, thanks!